src/fun_lti_provider/settings.py

```diff
@@ -0,0 +1,31 @@
+"""This module contains default settings for fun_lti_provider application"""
+
+from django.conf import settings
+
+# When a LTI launch request is successfully verified, the handler function defined
+# in the LTI_LAUNCH_SUCCESS_HANDLER setting is called with 2 parameters :
+# 1. The request (django.http.request.HttpRequest)
+# 2. The verifier LTI request (fun_lti_provider.lti.LTI)
+#
+# The handler should return a valid HTTP response (django.http.HttpResponse)
+# A default implementation is given in this application as an exemple
+#
+# If you use the default implementation, you must add the following
+# authentication # backend in your AUTHENTICATION_BACKENDS setting :
+#   "fun_lti_provider.default.backend.AuthBackend"
+#
+
+settings.LTI_LAUNCH_SUCCESS_HANDLER = getattr(
```

Comment:
It would be nice to define defaults in your app so that this setting is made optional (see marsha or richie for the pattern).

Reply:
I don't understand this comment, since the purpose of this module is precisely to provide default values for this application. I used [this pattern](https://stackoverflow.com/a/8429282).

I looked at richie's pattern with `defaults.py` files in apps and it seems to be almost the same.

Reply:
The difference is that the setting is defined in the defaults.py file from which the code imports it. So it still works if the project has no setting with this name at all.
Said differently, it's like proxy cache for the settings module :wink:

Reply:
We agree on the operating principe :slightly_smiling_face: 

And here, this is the same thing : 
- The [fun_lti_provider/settings.py](https://github.com/openfun/ashley/blob/fun_lti_provider/src/fun_lti_provider/settings.py) (that can be renamed to `defaults.py`) proxifies settings and add a default value if not defined.
- It is imported from the code that use these values (example in [views.py](https://github.com/openfun/ashley/blob/fun_lti_provider/src/fun_lti_provider/views.py#L10))
- The project works if no settings with this name at all is defined in the project : for example, these values are not defined in sandbox's [settings.py](https://github.com/openfun/ashley/blob/fun_lti_provider/sandbox/settings.py) and it works. 

And by writing this, I see that a test is missing for the `view.launch` to prove that it works  as described :slightly_smiling_face:

Reply:
ok sorry I just looked at the name and thought it was Django's settings file.